src/helpers/index.ts

```diff
@@ -15,6 +14,15 @@ export const formatPrice = (
   return amount;
 };
 
+export const formatMoney = (value?: Money | null): string => {
+  if (!value) return '0';
+
+  const priceValue = value as DineroObject;
+
+  const amount = priceValue ? Dinero(priceValue).toFormat('0.00') : '0';
```

Comment:
```suggestion
  const amount = value as DineroObject ? Dinero(priceValue).toFormat('0.00') : '0';
```

Type-casting isn't super pretty. We should probably have some util that converts `Money` to `DineroObject`, but this is fine for now :)